tests/ToastMessage.elm

```diff
@@ -0,0 +1,18 @@
+module ToastMessage exposing (suite)
+
+import Expect exposing (Expectation)
+import Fixture.ToastMessage exposing (toastFuzzer)
+import Json.Decode as Decode exposing (decodeValue)
+import Message.Toast as Toast
+import Test exposing (..)
+
+
+suite : Test
+suite =
+    describe "Toast Messages"
+        [ fuzz toastFuzzer "Encodes and decodes consistently" <|
+            \toast ->
+                Toast.encode toast
+                    |> decodeValue Toast.decoder
+                    |> Expect.equal (Ok toast)
+        ]
```

Comment:
This test generates 100 random instances of a `Toast`, round-trips them through JSON, and validates that the output matches the input, ensuring that decoding and encoding match up.